etcdserver/auth/auth_test.go

```diff
@@ -71,16 +86,16 @@ func TestMergeUser(t *testing.T) {
 			User{User: "foo", Roles: []string{"role1", "role2"}},
 			false,
 		},
-		{
-			User{User: "foo"},
-			User{User: "foo", Password: "$2a$10$aUPOdbOGNawaVSusg3g2wuC3AH6XxIr9/Ms4VgDvzrAVOJPYzZILa"},
-			User{User: "foo", Roles: []string{}, Password: "$2a$10$aUPOdbOGNawaVSusg3g2wuC3AH6XxIr9/Ms4VgDvzrAVOJPYzZILa"},
+		{ // empty password will not overwrite the previous password
```

Comment:
This test merges `tt.input` with `tt.merge`. So if we want to test `empty password will not overwrite the previous password`, the previous password must come from the one that gets overwritten, which is `tt.input.User`?

`tt.input` is the one that would get overwritten with previous implementation. Please correct me if I am wrong.